lang/java/reef-tang/tang/src/main/java/org/apache/reef/tang/examples/timer/TimerMock.java

```diff
@@ -56,8 +56,9 @@ public static void main(final String[] args) throws BindException, InjectionExce
   }
 
   @Override
-  public void sleep() {
-    System.out.println("Would have slept for " + seconds + "sec.");
+  public void sleep() throws InterruptedException {
+    java.lang.Thread.sleep(seconds * 1000);
```

Comment:
Do we need to sleep in TimerMock? Originally both README.md and TimerMock.java don't sleep, and it's mock, it's weird to both sleep and print "would have slept"